test/e2e/container_clone_test.go

```diff
@@ -267,4 +267,25 @@ var _ = Describe("Podman container clone", func() {
 		Expect(clone).ToNot(Exit(0))
 
 	})
+
+	It("podman container clone network passing", func() {
+		networkCreate := podmanTest.Podman([]string{"network", "create", "testing123"})
+		networkCreate.WaitWithDefaultTimeout()
+		defer podmanTest.removeNetwork("testing123")
+		Expect(networkCreate).To(Exit(0))
+		run := podmanTest.Podman([]string{"run", "-dt", "--network", "testing123", ALPINE})
```

Comment:
The test should perfom a network connect before the clone and then latter check the the same network is used for the cloned container.

Reply:
I will add that now, sorry I forgot we discussed that.